From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 27 Oct 2020 00:03:43 +0000
Subject: messageTray: Don't play sound when banners are disabled

When support for notification sounds was added, it made some sense
to keep the 'enable-sound' setting independent from 'show-banners'.

However that changed when 'show-banners' was rebranded in the UI as
"Do Not Disturb", as sounds are at least as disturbing as the banners.

(cherry picked from commit 709c226fc3f48e6a3f5175a49b09e15bdc2dc05b)

Bug: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/2873
Origin: upstream, 3.38.2
---
 js/ui/messageTray.js | 2 --
 1 file changed, 2 deletions(-)

diff --git a/js/ui/messageTray.js b/js/ui/messageTray.js
index d89af2b..7aac503 100644
--- a/js/ui/messageTray.js
+++ b/js/ui/messageTray.js
@@ -784,8 +784,6 @@ var Source = GObject.registerClass({
 
         if (this.policy.showBanners || notification.urgency == Urgency.CRITICAL)
             this.emit('notification-show', notification);
-        else
-            notification.playSound();
     }
 
     notify(propName) {
